app/models/item.rb

```diff
@@ -0,0 +1,6 @@
+class Item < ApplicationRecord
+  validates :title, presence: true
+  validates :description, presence: true
+
+  belongs_to :user
+end
```

Comment:
`belongs_to :user` is fine for now, but there is chance you'll need to make this more advanced IF you want to see all of the items you have already traded away. there are other patterns as well, but you should def go to Brianna's clinic tomorrow